### DISCUSSION TITLE
Disable RabbitMQ Clustering

### DISCUSSION THREAD

**Discussion by asadhasnain** [rabbitmq/rabbitmq-server#9441] [Other]
Is there a way to disable clustering in RabbitMQ?

Following communication port should not listen
**Interface: 127.0.0.1, port: 25672, protocol: clustering, purpose: inter-node and CLI tool communication**  

I just want to restrict the ports to following:

Interface: [::1], port: 5672, protocol: amqp, purpose: AMQP 0-9-1 and AMQP 1.0
Interface: 127.0.0.1, port: 5672, protocol: amqp, purpose: AMQP 0-9-1 and AMQP 1.0

**Comment by michaelklishin** [comment]
@asadhasnain [that port is also used by CLI tools](https://rabbitmq.com/networking.html#ports) so no, you cannot fully disable it. But in the specific case above, the port listener is bound to localhost anyway.
